src/app/components/multiple-draft-generator/multiple-draft-generator.component.html

```diff
@@ -1,10 +1,44 @@
 <div class="modal-body">
-  <h5>Languages without a Draft</h5>
+  <div class="btn-group btn-group-toggle toggle-drafts" data-toggle="buttons">
+    <label
+      class="btn {{
+        languageType === 'published' ? 'btn-success' : 'btn-secondary'
+      }}"
+    >
+      <input
+        type="radio"
+        name="languageType"
```

Comment:
I would probably name this something differently, `languageType` seems like an odd name

Reply:
I've removed these inputs, to simplify code. They were not needed, as I could just add an onClick onto the label.